src/cli/common.rs

```diff
@@ -382,11 +384,11 @@ pub(crate) fn list_targets(distributable: DistributableToolchain<'_>) -> Result<
                 .as_ref()
                 .expect("rust-std should have a target");
             if component.installed {
-                let _ = t.attr(term2::Attr::Bold);
-                let _ = writeln!(t, "{target} (installed)");
+                let _ = t.attr(terminalsource::Attr::Bold);
+                let _ = writeln!(t.lock(), "{target} (installed)");
```

Comment:
A note for future investigation .... is the `let _ =` needed still? I presume its discarding the fallible from the call, but we're in a function that returns Result, so `?` should be usable.

Reply:
I will take a look later.